manifests/m/MongoDB/Server/7.0.9/MongoDB.Server.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MongoDB.Server
+PackageVersion: 7.0.9
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ADDLOCAL=ServerService REBOOT=ReallySuppress
+UpgradeBehavior: uninstallPrevious
+ProductCode: '{67B9C053-A966-4ACE-877D-541777565BCA}'
+AppsAndFeaturesEntries:
+- DisplayName: MongoDB 6.0.6 2008R2Plus SSL (64 bit)

```

Comment:
```suggestion
- DisplayName: MongoDB 7.0.9 2008R2Plus SSL (64 bit)
```

Reply:
The problem was just that the display name had not been updated? The installer URL was pointing to the newest version of mongoDB server, was it the case?